docs/docs-components/Overview.js

```diff
@@ -0,0 +1,128 @@
+// @flow strict
+import { Fragment, useState, type Node } from 'react';
+import { Box, Fieldset, RadioButton, Flex, Text } from 'gestalt';
+import Page from './Page.js';
+import PageHeader from './PageHeader.js';
+import List from './OverviewList.js';
+import IllustrationContainer from './IllustrationContainer.js';
+import { type ListItemType } from './COMPONENT_DATA.js';
+
+export default function Overview({
+  buildingBlockComponents,
+  generalComponents,
+  platform,
+  utilityComponents,
+}: {|
+  buildingBlockComponents?: $ReadOnlyArray<ListItemType>,
+  generalComponents: $ReadOnlyArray<ListItemType>,
+  platform: 'Web' | 'Android' | 'iOS',
+  utilityComponents?: $ReadOnlyArray<ListItemType>,
+|}): Node {
+  const [order, setOrder] = useState('alphabetical');
+
+  const alphabeticalComponentList = [
+    ...(utilityComponents ?? []),
+    ...(buildingBlockComponents ?? []),
+    ...generalComponents,
+  ];
+
+  // GENERAL_COMPONENT_LIST is an array with component data. Each array item contains the SVG data and other metadata such as the component category. The following reduce method processes the GENERAL_COMPONENT_LIST array into an object grouping and mapping components per category so that we can map per category and pass each category value to <List />.
+  const GENERAL_COMPONENT_CATEGORY_MAP = generalComponents.reduce(
+    (acc, cur) => ({
+      ...acc,
+      // $FlowFixMe[invalid-computed-prop]
+      [cur.category]: (acc[cur.category] ?? []).concat(cur),
+    }),
+    {},
+  );
+
+  return (
+    <Page title={`${platform} component overview`} hideSideNav hideEditLink>
+      <Flex direction="column" width="100%">
+        <IllustrationContainer justifyContent="start">
+          <PageHeader
+            name={`${platform} component overview`}
+            description={`Gestalt provides an extensive set of React components for use in building larger web experiences and patterns. They include interactive UI components and developer utilities to help with implemention.
+
+Not sure which component to use? [Set up time with the Gestalt team.](/get_started/how_to_work_with_us#Slack-channels)`}
+            type="guidelines"
+          />
+        </IllustrationContainer>
+        <IllustrationContainer justifyContent="start">
+          <Flex
+            gap={{
+              row: 2,
+              column: 0,
+            }}
+            alignItems="center"
+          >
+            <Box aria-hidden>
+              <Text>Sort by</Text>
+            </Box>
+            <Fieldset
+              legend="Sort Gestalt components alphabetically or categorically"
+              legendDisplay="hidden"
+            >
+              <Flex
+                gap={{
+                  row: 2,
+                  column: 0,
+                }}
+              >
+                <RadioButton
```

Comment:
RadioGroup would be great!

Reply:
DONE

Reply:
<img width="818" alt="Screen Shot 2022-08-26 at 1 08 43 PM" src="https://user-images.githubusercontent.com/10593890/186891071-13e9e621-8719-4a76-97ea-2eb53351bb94.png">